common/core_api_review/kibana.api.ts

```diff
@@ -0,0 +1,72 @@
+// @public

```

Comment:
What is this core_api_review directory exactly? Just any "public" exported type? How does api-extractor/api-documentor use it? Would it make sense to just gitignore this?

Reply:
I'll expand a bit on how this works in the PR description, it's probably valuable context for anyone reviewing this.